Lib/extractor/formats/opentype.py

```diff
@@ -32,6 +34,23 @@ def isOpenType(pathOrFile):
         return False
     return True
 
+def isVarFont(pathOrFile):
+    try:
+        font = TTFont(pathOrFile)
+        fvar = varFont["fvar"]
```

Comment:
I think `varFont` should be `font` here and below.

Reply:
Oops, I just fixed this!